Client/MirScenes/SelectScene.cs

```diff
@@ -507,7 +551,7 @@ protected override void Dispose(bool disposing)
         #endregion
         public sealed class NewCharacterDialog : MirImageControl
         {
-            private static readonly Regex Reg = new Regex(@"^[A-Za-z0-9]{" + Globals.MinCharacterNameLength + "," + Globals.MaxCharacterNameLength + "}$");
+            private static readonly Regex Reg = new Regex(@"^[A-Za-z0-9ㄱ-힣]{" + Globals.MinCharacterNameLength + "," + Globals.MaxCharacterNameLength + "}$");
```

Comment:
no need for these special characters in the client - should be changed back to "[A-Za-z0-9]"